posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from scipy.interpolate import interp1d
</pre><p>Irrelevant for this blog but interp1d is now considered legacy: <a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp1d.html" target="_blank">https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp1d.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>